src/Processors/QueryPlan/MergingAggregatedStep.h

```diff
@@ -27,6 +27,8 @@ class MergingAggregatedStep : public ITransformingStep
     void describeActions(FormatSettings & settings) const override;
 
 private:
+    void updateOutputStream() override;
+
     AggregatingTransformParamsPtr params;
```

Comment:
Note that aggregation keys (and aggregates) from params can store *positions*.
If we implement such a method, we must check specified input stream was valid.
I think for some cases we can leave the method "Not implemented"